src/nuget/Microsoft.DotNet.BuildTools.VSTSBuildsApi.nuspec

```diff
@@ -0,0 +1,28 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<package>
+  <metadata minClientVersion="2.8.1">
+    <id>Microsoft.DotNet.BuildTools.VSTSBuildsApi</id>
```

Comment:
Nitpicky, but mind changing to `Microsoft.DotNet.BuildTools.VstsBuildsApi`, and changing filenames? I like package ID to match the namespace.